recipes/ligo-common/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.0.3" %}
+
+package:
+  name: ligo-common
+  version: {{version}}
+
+source:
+  fn: ligo-common-{{version}}.tar.gz
+  url: http://software.ligo.org/lscsoft/source/ligo-common-{{version}}.tar.gz
+  md5: 142c2e287cc09bd0192c916a0be6e492
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - setuptools
```

Comment:
Comes with `pip`. So we can leave this out.

Reply:
@jakirkham, is there any problem with being more verbose?

Reply:
Not particularly. Just a suggestion.